src/db.jl

```diff
@@ -23,19 +23,19 @@ end
 Base.getindex(db::DB, word::AbstractString, pos::Char) = db[pos, word]
 
 function load_lemmas(base_dir)
-    lemmas = Dict{Char, Dict{AbstractString, Lemma}}()
+    lemmas = Dict{Char, Dict{String, Lemma}}()
 
     for pos in ['n', 'v', 'a', 'r']
-        d = Dict{AbstractString, Lemma}()
-        
+        d = Dict{String, Lemma}()
+
         open(path_to_index_file(base_dir, pos)) do f
             for (i, line) in enumerate(eachline(f))
                 i > 29 || continue  # Skip Copyright.
-                word = line[1:(search(line, ' ')-1)]
+                word = line[1:(something(findfirst(isequal(' '), line), 0)-1)]
```

Comment:
I don't think we need the `something` on this like.
That is only there to handle if the thing is not found,
and we are sure it will be found

Reply:
Ok, done